content/_data/changelogs/lts.yml

```diff
@@ -7450,6 +7472,29 @@
       Wait for the computation to finish when triggering a new build while the build graph is being recomputed.
       This guarantees that recently updated build triggers are executed.
 
+  # Packaging
+
+  - type: rfe
+    category: rfe
+    authors:
+      - basil
+    pr_title: Add debian_slim_jdk17 image
+    references:
+      - url: https://github.com/jenkinsci/docker/pull/1360
+        title: Docker pull request 1360
+    message: |-
+      Add slim Debian-based JDK 17 Docker image.
+  - type: rfe
+    category: rfe
+    authors:
+      - basil
+    pr_title: Add alpine_jdk17
+    references:
+      - url: https://github.com/jenkinsci/docker/pull/1362
+        title: Docker pull request 1362
+    message: |-
+      Add Alpine-based JDK 17 Docker image.
```

Comment:
The most significant new features added to the packaging. I included these in a separate section at the end because Java 17 support is not as important as the compatibility problems presented earlier.